qutebrowser/config/qtargs.py

```diff
@@ -259,3 +259,10 @@ def init_envvars() -> None:
                    if qtutils.version_check('5.14', compiled=False)
                    else 'QT_AUTO_SCREEN_SCALE_FACTOR')
         os.environ[env_var] = '1'
+
+    for var in config.val.qt.environ:
+        val = config.val.qt.environ[var]
```

Comment:
```suggestion
    for var, val in config.val.qt.environ.items():
```

Reply:
I originally had written it the way you suggest however then I get the following error when doing the pylint test:

`qutebrowser/config/qtargs.py:263:20: E9998: qt.environ.items is no valid config option. (bad-config-option)`

so I switched it. Should I just ignore the pylint test or do I need to change something else as well?

Reply:
Oh, that's a bug in that test (written by me). Feel free to ignore it and I'll see what I can do to fix it.